src/mbi/estimation.py

```diff
@@ -393,15 +393,15 @@ def update(w, v, gbar, c, beta, t):
         w = (1 - c) * w + c * v
         return w, v, gbar
 
-    w = v = marginal_oracle(potentials, known_total, mesh)
+    w_var = v_var = marginal_oracle(potentials, known_total, mesh)
```

Comment:
Please make an exception for these single-letter variable names come from specific academic papers.  Revert the names to w, v and just disable pylint here